tests/Doctrine/Tests/DBAL/Functional/Driver/PDOConnectionTest.php

```diff
@@ -66,29 +70,31 @@ public function testThrowsWrappedExceptionOnExec()
 
     public function testThrowsWrappedExceptionOnPrepare()
     {
-        if ($this->connection->getDriver()->getName() === 'pdo_sqlsrv') {
+        $driver = $this->connection->getDriver();
+
+        if ($driver instanceof PDOSQLSRVDriver) {
             $this->markTestSkipped('pdo_sqlsrv does not allow setting PDO::ATTR_EMULATE_PREPARES at connection level.');
         }
 
+        // Some PDO adapters do not check the query server-side
+        // even though emulated prepared statements are disabled,
+        // so an exception is thrown only eventually.
+        if ($driver instanceof PDOOracleDriver
+            || $driver instanceof PDOPgSQLDriver
+        ) {
+            self::markTestSkipped(sprintf(
+                'The underlying implementation of the "%s" driver does not check the query to be prepared server-side.',
```

Comment:
The class name shouldn't be wrapped in double quotes right since it is a symbol?

Reply:
No, it shouldn't 😬

Reply:
hehe, nice. Makes me feel better about how many times I got it wrong in my previous PR where you had to point it out to me.